Catappult/appcoins-wallet-android
# [APPC-2233] - Remove logging when building release

**What does this PR do?**

Added product flavours to allow stop logging info and debug in production.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] LogInterceptor.java

**How should this be manually tested?**

Launch the wallet in flavour nologger. You should no longer see info and debug loggings in logcat. Namely, network calls.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-2233](https://aptoide.atlassian.net/browse/APPC-2233)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)

Flavours logging and nologger were created.


**Code Review Checklist**
